### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by hexflores87**
Model validations for recipes.

### CODE REVIEW

File: app/javascript/components/pages/AboutUs.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React, { Component } from "react";
+
+class AboutUs extends Component {
+  render() {
+    return <div></div>;
```
Comment (minachuong) [inline app/javascript/components/pages/AboutUs.js:5]: 👀

Comment (minachuong) [reply app/javascript/components/pages/AboutUs.js:5]: How is the AboutUs component being verified as rendering on App?

File: spec/models/recipe_spec.rb [hunk 2]
```diff
@@ -1,5 +1,38 @@
...
+#   * ½ pound ground beef 1 slice gluten free bread ⅓ cup milk Salt and black pepper to taste 1 egg yolk  ½ cup grated Parmigiano Reggiano cheese.", instructions: 'In a large bowl, combine gluten free bread and milk. Let bread soak, then squeeze out excess milk. In the same bowl, add bread, egg, ground beef, 1/2 the parsley, 1/2 the cheese, and season with salt and pepper. Roll into 3/4 inch balls &amp; set aside. In a large skillet, gently cook the garlic and chili flakes in olive oil. Add meatballs and cook thoroughly. Add the cherry tomatoes and saute for 2 minutes, then season with salt and pepper to taste. Deglaze with wine and reduce mixture by half. Cook the pasta according to package directions, then reserve 1/2 cup of the cooking liquid. Drain the pasta then toss with the sauce and add the reserved cooking liquid. Cook the pasta in the sauce for one minute, remove the skillet from the heat and fold in the remaining cheese and parsley.', ready_in_minutes: 25,cooking_minutes: 15,  
+#   expect(recipe.errors[:diet_type]).to_not be_empty
+#   end
```
Comment (minachuong) [inline spec/models/recipe_spec.rb]: It'd be good to uncomment this test to see if it's working

**Review by minachuong — APPROVED** [review]
Good clean up on making sure folders are nested in a predictable way. See comments

File: app/javascript/components/pages/AboutUs.js (hunk 1 above)
Comment (hexflores87) [reply app/javascript/components/pages/AboutUs.js:5]: We just started that when we were waiting for an Adult to help us.

File: spec/models/recipe_spec.rb (hunk 2 above)
Comment (hexflores87) [reply spec/models/recipe_spec.rb]: We just uncommented it and it is not working.

Comment (hexflores87) [reply spec/models/recipe_spec.rb]: jk it worked

File: app/javascript/components/pages/AboutUs.js (hunk 1 above)
Comment (minachuong) [reply app/javascript/components/pages/AboutUs.js:5]: 👍

File: spec/models/recipe_spec.rb [hunk 3]
```diff
@@ -1,5 +1,38 @@
...
+  * ½ pound ground beef 1 slice gluten free bread ⅓ cup milk Salt and black pepper to taste 1 egg yolk  ½ cup grated Parmigiano Reggiano cheese.", instructions: 'In a large bowl, combine gluten free bread and milk. Let bread soak, then squeeze out excess milk. In the same bowl, add bread, egg, ground beef, 1/2 the parsley, 1/2 the cheese, and season with salt and pepper. Roll into 3/4 inch balls &amp; set aside. In a large skillet, gently cook the garlic and chili flakes in olive oil. Add meatballs and cook thoroughly. Add the cherry tomatoes and saute for 2 minutes, then season with salt and pepper to taste. Deglaze with wine and reduce mixture by half. Cook the pasta according to package directions, then reserve 1/2 cup of the cooking liquid. Drain the pasta then toss with the sauce and add the reserved cooking liquid. Cook the pasta in the sauce for one minute, remove the skillet from the heat and fold in the remaining cheese and parsley.', ready_in_minutes: 25, diet_type: "Gluten-Free"
+  expect(recipe.errors[:cooking_minutes]).to_not be_empty
+end
+it "should have a diet_type" do
```
Comment (minachuong) [inline spec/models/recipe_spec.rb:33]: 👍

**Review by minachuong — APPROVED** [review]
👍 thanks for making fixes!

**Pull request merged**